src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -378,6 +390,111 @@
                 return this;
             },
 
+            loadToolbar: function(token) {
+                this.load(
+                    'sfwdt-container',
+                    '{{ path("_wdt_template") }}' + token,
```

Comment:
maybe generate the url with a placeholder token (`xxxxxx` or so( (thus previous route) and replace in JS.